### ISSUE TITLE
[Organization Signup] Errors on form does not get focus after first validation

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#6353]
## Area or Component

Organization Sign up - Organization information

## Quick Summary

Focus is not being sent to the first field with an error

### How to Replicate
1. Leave a required field empty
2. Click Continue
3. (Focus should be sent to required field)
4. Click Continue
5. Focus is not sent

## Accessibility Violation

[why is this inaccessible? include deque link if available.]: #

## Action Requested

- Use ReactHookForm for this form

## Screenshots (optional)
https://github.com/CDCgov/prime-simplereport/assets/10108172/54f38d38-04b2-4cd0-9b00-e2088f07c3ed

## Source Code (optional)

[which file or line is the issue attached to?]: #
